index.html

```diff
@@ -88,7 +95,8 @@ <h4 class="social-media-title">Let's Connect</h4>
           <div class="project-img">
             <a href="#">
               <img
-                src="images/about-image/tonic-project.png"
+                src="/images/about-image/Snapshoot_Portfolio.png"
+                class="project-1"
                 alt="tonic-project"
               />
             </a>
```

Comment:
- [ ] I noticed here for each card you added a border to it but I think it would be better if you remove it since it is not in the Figma design so kindly remove it

- [ ]  If you check your First card there is a background image but I think you don't need that since the card doesn't have any background image so please remove that to be the same with Figma 

- your design
![card1](https://user-images.githubusercontent.com/58553711/177783319-00d271f0-3904-428c-80b9-408854f91ab8.PNG)



- Figma design
![image](https://user-images.githubusercontent.com/58553711/177783184-809be560-5874-4854-9cbb-7e42385b5f8b.png)